src/ripple/protocol/impl/STObject.cpp

```diff
@@ -116,7 +116,8 @@ bool STObject::setType (const SOTemplate& type)
                 JLOG (debugJournal().warn())
                     << "setType(" << getFName().getName()
                     << "): explicit default " << e->e_field.fieldName;
-                valid = false;
+                Throw<std::runtime_error>("Explicit default " +
+                    e->e_field.fieldName);
```

Comment:
Thats unfortunate, and I understand if your position is that changing that is beyond the scope of this set of changes. However, we should not build on that mistake.